[rospy.client][INFO] 2021-09-09 17:19:31,912: init_node, name[/server_endpoint], pid[11978]
[xmlrpc][INFO] 2021-09-09 17:19:31,912: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-09-09 17:19:31,912: Started XML-RPC server [http://wng:36531/]
[rospy.init][INFO] 2021-09-09 17:19:31,913: ROS Slave URI: [http://wng:36531/]
[rospy.impl.masterslave][INFO] 2021-09-09 17:19:31,913: _ready: http://wng:36531/
[rospy.registration][INFO] 2021-09-09 17:19:31,913: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-09-09 17:19:31,913: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-09-09 17:19:32,013: registered with master
[rospy.rosout][INFO] 2021-09-09 17:19:32,013: initializing /rosout core topic
[rospy.rosout][INFO] 2021-09-09 17:19:32,014: connected to core topic /rosout
[rospy.simtime][INFO] 2021-09-09 17:19:32,014: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-09-09 17:19:32,021: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-09-09 17:19:32,257: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-09-09 17:20:12,030: Connection from 192.168.1.146
[rosout][INFO] 2021-09-09 17:20:12,034: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-09-09 17:20:12,065: topic[/execution_check] adding connection to [http://wng:36531/], count 0
[rospy.internal][INFO] 2021-09-09 17:20:12,066: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rosout][ERROR] 2021-09-09 17:51:58,838: Exception: No more data available
[rosout][INFO] 2021-09-09 17:51:58,839: Disconnected from 192.168.1.146
[rospy.core][INFO] 2021-09-09 17:52:51,518: signal_shutdown [signal-2]
[rospy.internal][ERROR] 2021-09-09 17:52:51,533: Unable to initiate TCP/IP socket to wng:40285 (http://wng:36531/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2021-09-09 17:52:51,533: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-09-09 17:52:51,534: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-09-09 17:52:51,534: topic[/execution_check] removing connection to http://wng:36531/
[rospy.impl.masterslave][INFO] 2021-09-09 17:52:51,535: signal-2
[rospy.core][INFO] 2021-09-09 17:52:52,152: signal_shutdown [atexit]
